src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -143,7 +148,7 @@ private void assertCommandFailure(String inputCommand, Class<? extends Throwable
     /**
      * Tests the Logic component's handling of an {@code IOException} thrown by the Storage component.
      *
-     * @param e the exception to be thrown by the Storage component
+     * @param e               the exception to be thrown by the Storage component
```

Comment:
same weird spacing here that can be removed.

Reply:
![image](https://github.com/user-attachments/assets/aced8615-0867-44a4-9d0c-8416897b1f9c)

Same thing here.